Cimpress.FulfillmentLocationNetCore/FulfillmentLocationClient.cs

```diff
@@ -83,6 +83,13 @@ private void _ConfigureClient()
             _httpClient.DefaultRequestHeaders.Add("UserAgent", "Fulfillment Location .NET Core Client by Cimpress");
         }
 
+        private string _GetAuthorization() {
```

Comment:
Is there a reason this starts with a capital letter? I don't know .NET's conventions.

Reply:
According to https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions:

> 

Method | Pascal | public class Object {   public virtual string ToString();  }
-- | -- | --